constraints.txt

```diff
@@ -3,6 +3,9 @@
 # https://github.com/Qiskit/qiskit-terra/issues/10345 for current details.
 scipy<1.11; python_version<'3.12'
 
+# Temporary pin to avoid CI issues caused by scipy 0.14.0
+scipy==1.13.1; python_version=='3.12'
```

Comment:
Can we make an issue in the tracker and link to it here, in case we forget to remove it in the future?

(Also, the problem is scipy 1.14.0 not 0.14)

Reply:
lol double whoops

Reply:
Thanks!